### PR TITLE
feat: turn on paginator when total Items are larger than page size

### PR DESCRIPTION
**Pull request by palbizu**
## Description
Turn on paginator when total items are greater than the smallest item in the page size drop-down menu.

### Testing
Visual testing

### Checklist:
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### Documentation
Make sure that you have documented corresponding changes in this repository or [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.

### CODE REVIEW

File: projects/components/src/paginator/paginator.component.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ import { PaginationProvider } from './paginator-api';
   styleUrls: ['./paginator.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <div class="paginator" *ngIf="this.totalItems">
+    <div class="paginator" *ngIf="this.totalItems && this.totalItems > pageSizeOptions[0]">
```
Comment (aaron-steinfeld) [inline projects/components/src/paginator/paginator.component.ts:22]: nit 
```suggestion
    <div class="paginator" *ngIf="this.totalItems && this.totalItems > this.pageSize">
```

Comment (palbizu) [reply projects/components/src/paginator/paginator.component.ts]: done

**Review by aaron-steinfeld — DISMISSED** [review]

**Review by aaron-steinfeld — DISMISSED** [review]

File: projects/components/src/paginator/paginator.component.test.ts [hunk 2]
```diff
@@ -110,4 +110,15 @@ describe('Paginator component', () => {
     expect(spectator.query(LabelComponent)?.label).toEqual('1-50 of 50');
     expect(spectator.component.hasNextPage()).toBe(false);
   });
+
+  test('should hide the pag when totalItems is less than the pageSizeOptions options', () => {
```
Comment (aaron-steinfeld) [inline projects/components/src/paginator/paginator.component.test.ts:114]:
```suggestion
  test('should hide the paginator when totalItems is less than the pageSizeOptions options', () => {
```

Comment (palbizu) [reply projects/components/src/paginator/paginator.component.test.ts]: done

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**